### PR TITLE
Add function call to cluster ignore_validation

### PR DESCRIPTION
**Pull request by bkopilov**
- Added set_ignored_validations
- Added get_ignored_validations

### CODE REVIEW

**Comment by bkopilov** [comment]
@lalon4 ,

**Comment by bkopilov** [comment]
@osherdp

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py [hunk 1]
```diff
@@ -223,6 +223,16 @@ def set_disk_encryption(self, mode: str, roles: str, tang_servers: str = None):
         disk_encryption_params = models.DiskEncryption(enable_on=roles, mode=mode, tang_servers=tang_servers)
         self.api_client.update_cluster(self.id, {"disk_encryption": disk_encryption_params})
 
+    def set_ignored_validations(self, host_validation_ids: List = None, cluster_validation_ids: List = None, **kwargs):
```
Comment (osherdp) [inline src/assisted_test_infra/test_infra/helper_classes/cluster.py:226]:
```suggestion
    def set_ignored_validations(self, host_validation_ids: list[str] = None, cluster_validation_ids: list[str] = None, **kwargs):
```

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py [hunk 2]
```diff
@@ -223,6 +223,16 @@ def set_disk_encryption(self, mode: str, roles: str, tang_servers: str = None):
...
+        ignore_obj = models.IgnoredValidations(
+            host_validation_ids=json.dumps(host_validation_ids) if host_validation_ids else None,
+            cluster_validation_ids=json.dumps(cluster_validation_ids) if cluster_validation_ids else None,
+        )
+        return self.api_client.client.v2_set_ignored_validations(self.id, ignore_obj, **kwargs)
+
+    def get_ignored_validations(self, **kwargs):
```
Comment (osherdp) [inline src/assisted_test_infra/test_infra/helper_classes/cluster.py:233]:
```suggestion
    def get_ignored_validations(self, **kwargs) -> <the returned type>:
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py (hunk 2 above)
Comment (bkopilov) [reply src/assisted_test_infra/test_infra/helper_classes/cluster.py:233]: ack

**Review by osherdp — APPROVED** [review]

**Pull request merged**